### DISCUSSION TITLE
Add the ability to tweak `RENOVATE_PERSIST_REPO_DATA` (or new variable) to only store `.git` folder

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#24858] [Suggest an Idea]
### What would you like Renovate to be able to do?

When using `RENOVATE_PERSIST_REPO_DATA` on ~20 few repos with Bundler and NPM installed, this leads to ~2GB repo cache.

Being able to tweak this to just cache the `.git` folder would _significantly_ reduce the cache size, while still containing benefit to reduce the `git fetch` time in the future.

### If you have any ideas on how this should be implemented, please tell us here.

Allow `RENOVATE_PERSIST_REPO_DATA=git` or a new environment variable `RENOVATE_PERSIST_GIT_DATA=true`

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by jamietanna** [comment]
Actually it may be possible by caching + restoring the `--base-dir`

**Comment by rarkins** [comment]
We'd need to know the exact commands you think would be required

**Comment by jamietanna** [comment]
This is very likely out of date from where I was originally looking at doing this, so closing
